test/e2e/specs/editor/various/change-detection.spec.js

```diff
@@ -295,12 +299,7 @@ test.describe( 'Change detection', () => {
 
 		await editor.canvas
 			.getByRole( 'textbox', { name: 'Add title' } )
-			.type( '!' );
-		await expect(
-			page
-				.getByRole( 'region', { name: 'Editor top bar' } )
-				.getByRole( 'button', { name: 'Save draft' } )
-		).toBeEnabled();
```

Comment:
The button isn't restored when making changes in flight. I'm not sure if this is a regression. The behavior can be reproduced manually by throttling the network via DevTools.